app.js

```diff
@@ -0,0 +1,7 @@
+let section = document.getElementsByTagName('section')[0];
```

Comment:
Since this didn't get reassigned, Can you change this to let?
```suggestion
const section = document.getElementsByTagName('section')[0];
```